capable/README.md

```diff
@@ -0,0 +1,21 @@
+# MobileCoin SGX: Capabilities
+
+[![Project Chat][chat-image]][chat-link]<!--
+-->![License][license-image]<!--
+-->![Target][target-image]<!--
+-->[![Crates.io][crate-image]][crate-link]<!--
+-->[![Docs Status][docs-image]][docs-link]<!--
+-->[![Dependency Status][deps-image]][deps-link]
+
+Idiomatic rust for the `sgx_capable` library.
+
+[chat-image]: https://img.shields.io/discord/844353360348971068?style=flat-square
+[chat-link]: https://mobilecoin.chat
+[license-image]: https://img.shields.io/crates/l/mc-sgx-capable?style=flat-square
+[target-image]: https://img.shields.io/badge/target-x86__64--unknown--linux--gnu-orange?style=flat-square
```

Comment:
:wrench: Should this be x86_64, it looks like capable is used for checking and enable SGX behavior so it's client side.  If it should be unknown then `capable/sys/README.md` should probably be updated n'sync.
```suggestion
[target-image]: https://img.shields.io/badge/target-x86__64-blue?style=flat-square
```